internal/ingress/controller/template/crossplane/http.go

```diff
@@ -284,6 +284,11 @@ func (c *Template) buildHTTP() {
 		httpBlock = append(httpBlock, buildMapDirective("$http_referer", "$block_ref", refDirectives))
 	}
 
+	for _, v := range cfg.CustomHTTPErrors {
+		httpBlock = append(httpBlock, buildDirective("error_page", v,
+			fmt.Sprintf("@custom_upstream-default-backend_%d", v)))
```

Comment:
using = as part of the argument should keep the consistencyu with the original template:
```golang
httpBlock = append(httpBlock, buildDirective("error_page", v,
			"=", fmt.Sprintf("@custom_upstream-default-backend_%d", v)))
```